### PR TITLE
Update clean part of the example doc

### PR DESCRIPTION
**Pull request by skycastlelily**
Pull Request Checklist

* [ ] implement the feature
* [ ] write the documentation
* [ ] extend the test coverage
* [ ] update the specification
* [ ] adjust plugin docstring
* [ ] modify the json schema
* [ ] mention the version
* [ ] include a release note

### CODE REVIEW

**Comment by skycastlelily** [comment]
This is really a tiny  merge request, which is pretty safe to be merged directly:)

**Review by therazix — APPROVED** [review]

File: docs/examples.rst [hunk 1]
```diff
@@ -1486,11 +1486,11 @@ Clean guests
 
 The subcommand ``clean guests`` aims to stop all running guests.
 By default, runs are taken from ``/var/tmp/tmt``, this can be
-changed using an argument to the subcommand:
+changed using ``--workdir-root`` to the subcommand:
```
Comment (happz) [inline docs/examples.rst:1489]: The sentence no longer sounds plausible, how about "changed using \`\`--workdir-root\`\` option of the subcommand:"?

File: docs/examples.rst [hunk 2]
```diff
@@ -1539,11 +1539,11 @@ Clean workdirs
 
 The goal of ``clean runs`` is to remove workdirs of past runs.
 Similarly to above, ``/var/tmp/tmt`` is used by default as run
-location and this can be changed using an argument:
+location and this can be changed using ``--workdir-root``:
```
Comment (happz) [inline docs/examples.rst:1542]: How about "... using \`\`--workdir-root\`\` option:`"

**Comment by skycastlelily** [comment]
Updated^^

**Review by happz — APPROVED** [review]

**Comment by happz** [comment]
Documentation-only change, not waiting for full functional tests.

**Pull request merged**